PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -99,7 +99,11 @@ extension DefaultControlsViewController {
             
             loading = props.player?.item.playable?.loading ?? false
             
-            playButtonHidden = props.player?.item.playable?.playbackAction.play == nil || controlsHidden
+            playButtonHidden = {
+                let playbackAcionNotPlay = props.player?.item.playable?.playbackAction.play == nil
```

Comment:
`playbackAcionNotPlay`? maybe better `emptyPlaybackAcion`

Reply:
@BogdanBilonog well yeah, it sounds better.